x/concentrated-liquidity/swaps_test.go

```diff
@@ -624,36 +643,52 @@ var (
 		//          5000
 		//  4545 -----|----- 5500
 		//  4545 -----|----- 5500
-		"two positions within one tick: usdc -> eth": {
-			tokenOut:                 sdk.NewCoin("usdc", sdk.NewInt(42000000)),
+		"two positions within one tick: eth (in) -> usdc (out) zfo": {
```

Comment:
nit: put `zfo` and `ofz` in square brackets or somehow separate them to make it clear they're intentional (thought they were typos at first)

Reply:
Done